diff --git a/dist/provider/base.js b/dist/provider/base.js
index d0624b1b88c328883d72baddc46a4bd79d254878..5ac64e42a4f256fa984370414802447870346572 100644
--- a/dist/provider/base.js
+++ b/dist/provider/base.js
@@ -33,7 +33,7 @@ function I18nProvider({ locales = [], locale, onLocaleChange, children, translat
         else {
             segments[0] = value;
         }
-        router.push(`/${segments.join('/')}`);
+        location.href = `/${segments.join('/')}`;
     };
     const onChangeRef = useRef(onChange);
     onChangeRef.current = onChange;
